Gemfile

```diff
@@ -33,8 +33,8 @@ gem 'redis', '~> 3.0'
 # Use Capistrano for deployment
 # gem 'capistrano-rails', group: :development
 gem 'responders'
+gem 'bootstrap', '~> 4.0.0.beta2.1'
 gem 'jquery-rails'
-gem 'bootstrap-sass'
```

Comment:
Any reason why `bootsrap-sass` didn't do the trick? I believe @DominikSchmidtlein added that gem for the exact purpose of having bootstrap

Reply:
bootstrap-sass is the port for bootstrap 2 & 3, I updated it to use version 4 (different styling / components)